recipes/kraken-ea/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#export CPPFLAGS="-I$PREFIX/include"
+#export LDFLAGS="-L$PREFIX/lib"
+#export CPATH=${PREFIX}/include
+
+mkdir -p "$PREFIX/libexec" "$PREFIX/bin"
```

Comment:
I've not seen `libexec` before and I don't see it in the [FSH](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard). Would it be possible to use `$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM` instead?